src/workspace.cpp

```diff
@@ -14,6 +15,13 @@ QList<Workspace> Workspace::filterByCurrentScreen(QList<Workspace> inputWorkspac
     return outputWorkspaces;
 }
 
+QList<Workspace> Workspace::orderByName(QList<Workspace> inputWorkspaces) {
+    std::sort(inputWorkspaces.begin(),inputWorkspaces.end(),[](Workspace a,Workspace b){
+            return (a.id<b.id);  //NOTICE as the variables are named right now the full name is copied on the id!
```

Comment:
I am not sure I understand the comment here. Is it trying to explain why we are using the `id`s instead of the `name`s? If so, I am still not sure I understand, sorry... What am I missing?

Reply:
I think until now I was missing some terminology on this project, but I resolved it. Let me recap:
So with i3 configs looking like that `set $ws1 "Index:Name:Emoji"` 
and the five lines of code 
```c++
            auto splitName = QString::fromStdString(i3workspace->name).split(':');
            workspace.id = QString::fromStdString(i3workspace->name);
            workspace.index = splitName[0];
            workspace.name = splitName.size() == 1 ? splitName[0] : splitName[1];
            workspace.icon = splitName.size() == 3 ? splitName[2] : "";
```
You clearly mean that name is not the full name of the workspace, but just the substring "Name" from the string "Index:Name:Emoji" that is the full name of the workspace. Now I understand that, so I should rephrase what I implemented. It is not "sort by name" it is "sort by full i3 workspace name" which is accessible by the variable named `id`.